k8s/cronjob-reboot/cronjob.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: restart-deployment
+  namespace:  palworld
+spec:
+  concurrencyPolicy:  Forbid
+  schedule: '30 9 * * *'
+  jobTemplate:
+    spec:
+      backoffLimit: 1
+      activeDeadlineSeconds:  600
+      template:
+        spec:
+          serviceAccountName: restart-deploy
+          restartPolicy:  Never
+          containers:
+            - name: kubectl
+              image:  bitnami/kubectl
+              command:
+              - /bin/sh
+              - -c
+              - /restart-script/restart-deployment.sh
```

Comment:
do you even need a CM with the shell script? what about inline?

Reply:
That is a good point, I'm not sure actually. Pardon my ignorance, I'm a little new to this - when you say inline, do you mean taking the contents of the shell script and making it a one-liner instead of using a CM and mounting it as a volume?

Reply:
Hi sorry for the confusion. What i mean is instead of having a configmap mounted you could also put that script in that command as an arg like

```
command:
  - /bin/sh
  - -c
args:
  - |
    cont=$(kubectl -n palworld get pods -o=jsonpath='{.items[?(@.metadata.labels.app=="palworld-server")].metadata.name}')
    kubectl exec -n palworld -i pod/$cont rcon-cli "Broadcast Saving_Server_Data..."
    kubectl exec -n palworld -i pod/$cont rcon-cli save
    sleep 30
    kubectl exec -n palworld -i pod/$cont rcon-cli "Broadcast Backing_up_Server_Data..."
    kubectl exec -n palworld -i pod/$cont backup
 ...
```